src/main/java/net/glowstone/generator/TheEndGenerator.java

```diff
@@ -121,26 +125,26 @@ private ChunkData generateRawTerrain(World world, int chunkX, int chunkZ) {
     private void generateTerrainDensity(World world, int x, int z) {
         Map<String, OctaveGenerator> octaves = getWorldOctaves(world);
         double[] roughnessNoise = ((PerlinOctaveGenerator) octaves.get("roughness"))
-            .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
+                .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
         double[] roughnessNoise2 = ((PerlinOctaveGenerator) octaves.get("roughness2"))
-            .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
+                .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
         double[] detailNoise = ((PerlinOctaveGenerator) octaves.get("detail"))
-            .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
+                .getFractalBrownianMotion(x, 0, z, 0.5D, 2.0D);
 
         int index = 0;
 
         for (int i = 0; i < 3; i++) {
             for (int j = 0; j < 3; j++) {
-                double nH = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
-                nH = Math.max(-100.0D, Math.min(80.0D, nH));
+                double nh = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
```

Comment:
Rename it to noiseHeight? That's my best guess on what this var should be described as.

Reply:
Done.